### PR TITLE
[1LP][RFR] workaround for server name in 5.10

### PR DESCRIPTION
**Pull request by izapolsk**
There is an issue in new appliances 5.9/5.10 appliances when server name is displayed wrongly.
This is temporary fix for tests which fail due to that issue.
REQUIRES: https://github.com/RedHatQE/widgetastic.core/pull/137

{{ pytest: --long-running --use-template-cache cfme/tests/integration/test_cfme_auth.py cfme/tests/intelligence/chargeback/test_chargeback_long_term_rates.py cfme/tests/v2v/test_v2v_migrations_ui.py cfme/tests/services/test_service_rbac.py cfme/tests/configure/test_access_control.py cfme/tests/intelligence/reports/test_validate_chargeback_report.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6444f0a2de26d48a61de2b18ad52f15bd18ae119:

`cfme/base/ui.py`:
- :bangbang: Line 791:31: **E131** *continuation line unaligned for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2cfe257f9ca8b1b3be1406e2277a4c6f02fd8509:

`cfme/base/ui.py`:
- :bangbang: Line 791:31: **E131** *continuation line unaligned for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/base/ui.py [hunk 1]
```diff
@@ -40,6 +40,14 @@ def address(self):
     return self.appliance.url
 
 
+def prepare_server_name(name):
+    # there is a bug in CF 5.10 UI when it has different number of spaces in different html attrs
+    # this is temporary workaround until this BZ is fixed
+    new_name = re.escape(name)
```
Comment (mshriver) [inline cfme/base/ui.py]: BZ please, at least in comment, maybe even use BZ().blocks to control flow here so we automatically stop working around it?

Comment (izapolsk) [reply cfme/base/ui.py]: Of course. I haven't submitted it yet.  I was working on workaround.

File: cfme/base/ui.py [hunk 2]
```diff
@@ -493,19 +498,23 @@ class advanced(WaitTab):  # noqa
...
-            "Zones",
-            "Zone: {} (current)".format(self.context['object'].zone.description),
-            "Server: {name} [{sid}]{current}".format(
+        if not (self.in_configuration and self.accordions.settings.is_displayed):
+            return False
+
+        currently_selected = self.accordions.settings.tree.currently_selected
+        if not ({self.context['object'].zone.region.settings_string, "Zones",
+                 "Zone: {} (current)".format(self.context['object'].zone.description)} <=
+                set(currently_selected)):
```
Comment (mshriver) [inline cfme/base/ui.py]: Please write some in-line comments here describing your intent. not clear to me why you're comparing the first few components of the Zones match to currently_selected wrapped in a set.

Comment (izapolsk) [reply cfme/base/ui.py]: okay, I just made we same as it was in before with only difference - re.match call

**Comment by izapolsk** [comment]
this is definitely temporary fix because it seems server name is screwed up either by automation or during appliance deployment.

**Comment by dajoRH** [comment]
Lint report for commit 5cf0eeb93fbf8928f887c239b45a78987f9ad4be:

`cfme/utils/browser.py`:
- :bangbang: Line 272:101: **E501** *line too long (108 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/configure/configuration/server_settings.py [hunk 3]
```diff
@@ -703,7 +703,8 @@ def configure(self, auth_mode=None, auth_provider=None, user_type=None, reset=Fa
             logger.info('Authentication form reset, returning')
             return
         elif changed:
-            if validate and mode not in [AUTH_MODES['database'], AUTH_MODES['external']]:
+            if validate and mode not in [AUTH_MODES['database'], AUTH_MODES['external'],
```
[the remark this answers is not in the document]
Comment (izapolsk) [reply cfme/configure/configuration/server_settings.py]: there is no Valid button in these views.

**Comment by mshriver** [comment]
Travis failed lint because #8401 is still open.

**Pull request merged**